l10n_it_delivery_note/tests/test_stock_delivery_note.py

```diff
@@ -101,14 +99,9 @@ def test_delivery_without_so(self):
         picking.button_validate()
 
         # create delivery note with advanced mode
-        dn_form = Form(
-            self.env["stock.delivery.note.create.wizard"].with_context(
-                {"active_ids": [picking.id]}
-            )
-        )
-        dn = dn_form.save()
-        dn.confirm()
-        self.assertTrue(picking.delivery_note_id)
-        picking.delivery_note_id.action_confirm()
-        self.assertEqual(picking.delivery_note_id.state, "confirm")
-        self.assertEqual(picking.delivery_note_id.invoice_status, "no")
+        dn = picking.delivery_note_id
+
+        self.assertTrue(dn)
+        dn.action_confirm()
+        self.assertEqual(dn.state, "confirm")
+        self.assertEqual(dn.invoice_status, "no")
```

Comment:
Questa parte di codice letteramente non fa quanto dice il commento... dovrebbe creare un DN in modalità avanzata, via wizard.

Reply:
Hai ragione, effettivamente non avevo visto quel commento perchè mi sono affidato solamente al primo alla riga 71 in cui si dice "_change user in order to automatically create delivery note when picking is validated_" che è un comportamento dell'impostazione "base".
Quale dei 2 commenti è preferibile seguire ai fini dei test?

Reply:
Grazie Marco @TheMule71, ho corretto.
Ho lasciato il metodo `test_partial_delivering_single_so` con le impostazioni base ed ho modificato `test_delivery_without_so` per usare il wizard.
Penso che il malinteso sia nato da dei copia/incolla che erano stati fatti tra i 2 metodi e che hanno lasciato il commento 
```
# change user in order to automatically create delivery note
# when picking is validated
```
nella seconda funzione